cmd/operator-sdk/generate/csv.go

```diff
@@ -124,19 +132,19 @@ func (c csvCmd) run() error {
 
 	// Write CRD's to the new or updated CSV package dir.
 	if c.updateCRDs {
-		crdManifestSet, err := findCRDs(c.includePaths...)
+		// TODO: Any requirement to update CRD's from any place other
+		// than the input "deploy" directory?
+		// Can CRD manifests be present outside of the "deploy" directory?
```

Comment:
@estroz

Reply:
> They will be present in the filepath.Join(c.inputs[gencatalog.DeployDirKey], "crds") dir, and not elsewhere

Since we're doing a `filepath.Walk` in `GetCRDManifestPaths()` it looks like the CRD manifests just need to be present under `c.inputs[gencatalog.DeployDirKey]` i.e `"deploy/..."` or under any subdirectory in that `filepath.Join(c.inputs[gencatalog.DeployDirKey], "foobar")` i.e `"deploy/foobar/..."`

But you're right, we should allow sourcing CRDs for CSV generation from any path.

Do we need a direct input (`--input crds=production/my-app-crds`) for this now to keep the functionality from CSV config file? 
Or is this something we can add in later on and I can put a TODO for this?

Reply:
We can add it in a follow-up.